src/middleware/fhir/base/base.service.js

```diff
@@ -82,6 +65,29 @@ module.exports.batch = (req, res) => new Promise((resolve, reject) => {
       return err;
     }));
   });
+  return {requestPromises, results};
+};
+
+module.exports.batch = (req, res) => new Promise((resolve, reject) => {
```

Comment:
Batch & transaction functions look identical other than the type. Can we unify them?